smoke-testing/src/test/java/games/strategy/engine/data/GameSaveTest.java

```diff
@@ -68,7 +69,7 @@ void testSaveGame(final String mapName, final String mapXmlPath) throws Exceptio
     final ServerGame game = startGameWithAis(gameSelector);
     final Path saveFile = Files.createTempFile("save", GameDataFileUtils.getExtension());
     game.saveGame(saveFile);
-    assertThat(Files.size(saveFile), is(not(0)));
+    assertThat(saveFile.toFile(), is(not(aFileWithSize(0))));
```

Comment:
:+1:   Does read really well.

There is a lot of room for us to create `FileMatchers` with our own match criteria if we so want.